### PR TITLE
Spike/call-psc-verif-api-securely

### PR DESCRIPTION
**Pull request by hepsimo**
- remove unused full-record page and its routers/handler
- save skeleton verification in confirmCompanyRouter via API
- behold verification record added to `transaction_verify` database

Resolves:
IDVA3-1127

### CODE REVIEW

**Comment by ch-code-analysis** [comment]
CI: No security warnings found

File: src/app.ts [hunk 1]
```diff
@@ -50,7 +50,7 @@ app.enable("trust proxy");
 
 // parse body into req.body
 app.use(express.json());
-app.use(express.urlencoded({ extended: false }));
+app.use(express.urlencoded({ extended: true }));
```
Comment (PhilipGlover) [inline src/app.ts:53]: is this still needed?

File: src/constants.ts [hunk 2]
```diff
@@ -25,6 +26,7 @@ export const PrefixedUrls = {
     COMPANY_NUMBER: servicePathPrefix + Urls.COMPANY_NUMBER,
     CONFIRM_COMPANY: servicePathPrefix + Urls.CONFIRM_COMPANY,
     INDIVIDUAL_PSC_LIST: servicePathPrefix + Urls.INDIVIDUAL_PSC_LIST,
+    FULL_RECORD: servicePathPrefix + Urls.FULL_RECORD, // temp. path to spike's data input screen
```
Comment (PhilipGlover) [inline src/constants.ts:29]: ditto

File: src/app.ts (hunk 1 above)
Comment (hepsimo) [reply src/app.ts:53]: Agreed, i think this should revert to false as there's no need.

**Review by PhilipGlover — APPROVED** [review]
- have successfully run code and PSC Verify resource created
- code looks good to me
Screenshot 2024-04-02 at 12 10 23 PM

**Pull request merged**